sites/dotcom/document.js

```diff
@@ -15,8 +18,36 @@ type Props = {
 
 export default ({ Page, data: { body, ...data } }: Props) => {
     const cleanedData = { ...parseCAPI(body), ...data };
-    return document({
-        Page: () => <App data={{ ...cleanedData }} Page={Page} />,
-        data: cleanedData,
+    const bundle = [
+        'javascript',
+        `${cleanedData.site}.${cleanedData.page.toLowerCase()}.js`,
+    ];
+    const MyPage = () => <App data={{ ...cleanedData }} Page={Page} />;
```

Comment:
Can we choose a better name than `MyPage`? Or maybe inline it into the line below?